src/js/common/schemaform/definitions/dateRange.js

```diff
@@ -3,17 +3,23 @@ import * as date from './date';
 
 import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
-export const schema = commonDefinitions.dateRange;
+export const schema = {
+  type: 'object',
+  properties: {
+    from: commonDefinitions.date,
+    to: commonDefinitions.date
+  }
+};
```

Comment:
I'm guessing the purpose of this just to simplify the schema? I.e., replace
```
dateRange: {
  from: {
    date: <start-date>
  },
  to: {
    date: <end-date>
  }
}
```
with
```
dateRange: {
  from: <start-date>,
  to: <end-date>
}
```
Any way to do this in the dateRange definition in definitions.js?